Products/ZenEvents/EventClass.py

```diff
@@ -369,18 +386,16 @@ def reIndex(self):
             for ip in evtclass.instances():
                 ip.index_object()
 
-
     security.declareProtected(ZEN_ADD, 'createCatalog')
     def createCatalog(self):
         """Create a catalog for EventClassRecord searching"""
         from Products.ZCatalog.ZCatalog import manage_addZCatalog
         manage_addZCatalog(self, self.default_catalog,
-                            self.default_catalog)
+                           self.default_catalog)
```

Comment:
I don't think this needs to be two lines.  It looks likes this can be all one line and still be under 80 characters, e.g.
```
       manage_addZCatalog(self, self.default_catalog, self.default_catalog)
```

Reply:
Indeed, fixed.